bundles/org.openhab.core/src/main/java/org/openhab/core/internal/items/ExpireManager.java

```diff
@@ -330,14 +334,8 @@ public ExpireConfig(Item item, String configString, Map<String, Object> configur
                     ? configString.substring(commaPos + 1).trim()
                     : null;
 
-            Object ignoreStateUpdatesConfigObject = configuration.get(CONFIG_IGNORE_STATE_UPDATES);
-            if (ignoreStateUpdatesConfigObject instanceof String) {
-                ignoreStateUpdates = Boolean.parseBoolean((String) ignoreStateUpdatesConfigObject);
-            } else if (ignoreStateUpdatesConfigObject instanceof Boolean) {
-                ignoreStateUpdates = (Boolean) ignoreStateUpdatesConfigObject;
-            } else {
-                ignoreStateUpdates = false;
-            }
+            ignoreStateUpdates = getBooleanConfigValue(configuration, CONFIG_IGNORE_STATE_UPDATES);
```

Comment:
```suggestion
            ignoreStateUpdates = ConfigParser.valueAsOrElse(configuration.get( CONFIG_IGNORE_STATE_UPDATES), Boolean.class, false);
```

We now have a core-method for that.

Reply:
`ConfigParser` is defined in `org.openhab.core.config.core`. Is it possible to add it as dependency to `org.openhab.core` (circular dependency)?

Reply:
No, unfortunately not. Keep your method then.